### DISCUSSION TITLE
What does FragDataBlockAuthReq do

### DISCUSSION THREAD

**Discussion by oedze** [Lora-net/LoRaMac-node#1200] [Q&A] [answered]
In the FUOTA example program the "FragDataBlockAuthReq" is sent during filetransfer. 

What does this command do? it is sent on the 201 port for the fragmentation package. 
I can't find it in the Lora fragmented block transport specification or anywhere else

**Comment by mluis1** [comment, marked as answer]
This request was part of the original non public LoRa-Alliance test specification.
This was used to verify that the downloaded binary file was correct.
